### DISCUSSION TITLE
Помогите пожалуйста. Не могу попасть в веб-интерфейс.

### DISCUSSION THREAD

**Discussion by Rognar92** [Anonym-tsk/nfqws-keenetic#130] [Q&A]
Пробовал порты 90 и 88, ввожу логин и пароль, в ответ просто очищение строк

**Comment by Anonym-tsk** [comment]
Значит неправильные логин/пароль вводите

**Reply by Rognar92** [reply]
Я копипастой мажу, той же самой, которой мажу веб-интерфейс роутера

**Reply by livebreakz** [reply]
так неправильно же. root:keenetic

**Reply by reanimatorpavel** [reply]
так неправильно же. root:keenetic - это как понимать?

**Reply by livebreakz** [reply]
это дефолтные логин и пароль

**Reply by reanimatorpavel** [reply]
Так всётаки... Как попасть в веб-интерфейс? Какой логин и пароль? Ничего из вышеперечисленного не работает.

**Comment by Rognar92** [comment]
попробовал разные варианты связки логин\пароль, решил переустановить. При переустановке выдало такой выхлоп

```
Configuring nfqws-keenetic-web.
 Starting /opt/sbin/lighttpd...              **failed**.
 Sending SIGHUP to /opt/sbin/lighttpd...
NFQWS Web Interface installed: http://192.168.1.1:90
```
Не помню, была и ли такая ошибка при первой установке.
Устанавливал из терминала (от админа) винды 11.

При этом окно логин\пароля всё так есть
в /var/log вообще пусто

**Comment by reanimatorpavel** [comment]
Также не могу попасть в веб-интерфейс. Логин и пароль entware не подходит.

**Comment by PhotoSCH** [comment]
В инструкции [**Подготовка Keenetic/Netcraze**](https://github.com/Anonym-tsk/nfqws-keenetic?tab=readme-ov-file#%D0%BF%D0%BE%D0%B4%D0%B3%D0%BE%D1%82%D0%BE%D0%B2%D0%BA%D0%B0-keeneticnetcraze) сбивает с толку фраза "логин - root, пароль по умолчанию - keenetic". Эта пара логин-пароль появится и нужна будет после выполнения инструкций **[Установка на Keenetic/Netcraze...](https://github.com/Anonym-tsk/nfqws-keenetic?tab=readme-ov-file#%D1%83%D1%81%D1%82%D0%B0%D0%BD%D0%BE%D0%B2%D0%BA%D0%B0-%D0%BD%D0%B0-keeneticnetcraze-%D0%B8-%D0%B4%D1%80%D1%83%D0%B3%D0%B8%D0%B5-%D1%81%D0%B8%D1%81%D1%82%D0%B5%D0%BC%D1%8B-%D1%81-entware)**. 
На этапе подготовки нужно войти по telnet  используя логин-пароль, по которым вы обычно входите в веб интерфейс роутера, например _**admin-ваш пароль**_, а потом ввести команду _**exec sh**_. 
Проверено на трех роутерах, два из которых старенькие ZyXEL KEENETIC.
![2025-02-13_182000](https://github.com/user-attachments/assets/197e83c9-4818-4f23-a14e-eae1136c360d)

**Reply by Zik** [reply]
Тоже была проблема с авторизацией, при чем вводил все верно. Пока не попробовал логин **root** (админ вообще не подходил, при условии что у меня ещё у пары логинов доступ есть), а пароль уже от cli интерфейса. Мистика
